test/e2e/storage/drivers/in_tree.go

```diff
@@ -1269,6 +1268,39 @@ func InitGcePdDriver() testsuites.TestDriver {
 	}
 }
 
+// InitWindowsGcePdDriver returns gcePdDriver running no Windows cluster that implements TestDriver interface
+func InitWindowsGcePdDriver() testsuites.TestDriver {
+	// In current test structure, it first initialize the driver and then set up
+	// the new framework, so we cannot get the correct OS here. So here set to
+	// support all fs types including both linux and windows. We have code to check Node OS later
```

Comment:
You may also want to update comments for these.

Reply:
updated. thanks!